app/src/pages/admin/serverSettingsPage/serverSettingsTabs/authConfigurationTab/forms/ssoUsersForm/ssoUsersForm.jsx

```diff
@@ -65,8 +82,27 @@ const SsoUsersFormComponent = ({ enabled: enabledFromStore, fetchAppInfo }) => {
   const getDescription = () =>
     formatMessage(enabled ? messages.ssoOnlyDescription : messages.manualInvitesDescription);
 
-  const handleToggle = (value) => {
-    setEnabled(value);
+  const handleToggle = async (value) => {
+    try {
+      await fetch('/api/v1/settings', {
```

Comment:
All urls should be used from `common/utils/URLS`.

Reply:
Done!